2024-02-16 15:12:47.463864+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 15:12:47.467865+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 15:12:47.482863+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 15:12:47.483863+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 14:13:57 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-02-16 14:14:03 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-02-16 23:48:08.647062+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:08.653538+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:08.665094+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:08.667313+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:18.562777+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:18.565278+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:18.568291+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:48:18.569291+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:51:50.023517+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:51:50.030164+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:51:50.045192+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:51:50.047158+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:52:18.220393+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:53:51.814834+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:53:51.821285+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:53:51.836324+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:53:51.838336+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:53:58.196940+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:55:05.583322+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:55:05.584833+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:55:05.587844+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-16 23:55:05.587844+01:00 jdbc[25]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:02:42.298112+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:02:42.304116+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:02:42.317376+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:02:42.318383+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:02:57.637950+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:08.393639+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:08.398169+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:08.403168+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:08.404168+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:46.870427+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:46.875946+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:46.889030+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:46.889995+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:05:57.074294+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:11:18.469171+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:11:18.476719+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:11:18.491240+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:11:18.492245+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:11:19.059091+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:12:32.551415+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:12:32.559777+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:12:32.581385+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:12:32.582383+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:12:55.125016+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:13:30.067459+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:13:30.073518+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:13:30.090117+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:13:30.091103+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:13:39.965044+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:43.465028+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:43.469552+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:43.473551+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:43.475060+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:46.089239+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:46.092243+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:46.094242+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:14:46.095243+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:15:00.660690+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:15:00.663722+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:15:00.667245+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:15:00.669242+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:16:07.683618+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:16:07.689198+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:16:07.705006+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:16:07.705006+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:14.409200+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:14.415195+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:14.429885+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:14.430887+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:35.727469+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:52.263176+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:52.265691+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:52.268719+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:18:52.269701+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:34.954108+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:34.960626+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:34.973146+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:34.975137+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:48.254277+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:48.259716+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:48.273238+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:20:48.274241+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:21:00.933635+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:14.611783+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:14.617309+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:14.629372+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:14.630376+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:18.366587+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:18.369588+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:18.372589+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:18.373587+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:19.110652+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:33.109376+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:33.112366+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:33.114410+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:33.115366+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:36.919275+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:36.922273+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:36.924276+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:22:36.925276+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:24:00.296123+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:24:00.302228+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:24:00.316556+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:24:00.317551+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:24:07.387191+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:25:27.325708+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:25:27.331230+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:25:27.342787+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:25:27.343789+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:24.949365+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:24.958374+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:24.979432+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:24.980950+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:30.339348+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:30.342396+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:30.345356+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:30.346358+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:27:30.829150+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:28:50.421218+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:28:50.427741+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:28:50.443549+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:28:50.445546+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:28:58.809228+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:29:23.461413+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:29:23.466412+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:29:23.479157+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:29:23.480167+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:29:54.040678+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:30:37.797519+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:30:37.800038+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:30:37.803042+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:30:37.804041+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:32:44.876933+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:32:44.883444+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:32:44.904055+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:32:44.905053+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:32:53.906043+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:33:22.303999+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:33:22.305999+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:33:22.309053+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:33:22.309053+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:33:32.160227+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:41.191716+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:41.197752+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:41.209369+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:41.212600+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:44.974265+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:44.979263+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:44.982629+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:44.983629+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-02-17 00:35:45.515291+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-07 22:43:33.792445+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-07 22:43:33.817897+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-07 22:43:55.883064+02:00 database: wrong user or password; user: "AKOROVAI"
org.h2.message.DbException: Nieprawidłowy użytkownik/hasło
Wrong user name or password [28000-224]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.server.TcpServerThread.run(TcpServerThread.java:175)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nieprawidłowy użytkownik/hasło
Wrong user name or password [28000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:522)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 8 more
2024-05-07 22:44:31.709638+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-07 22:44:31.714936+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-10 12:29:40.792953+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-10 12:29:40.798955+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:09:41.119728+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:09:41.126304+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:15:14.073851+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:15:14.079463+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:15:38.264339+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:15:38.267338+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:17:17.714358+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:17:17.718361+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:17:25.078642+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:17:25.080688+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:19:45.292209+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:19:45.294459+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:30:00.260817+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:30:00.262860+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:30:43.601418+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:30:43.604250+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:31:33.183213+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:31:33.185733+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:32:26.186795+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:32:26.194309+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:33:37.544761+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:35:04.012525+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:35:04.015035+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:35:11.721094+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:35:11.725748+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:38:39.860839+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:38:39.863837+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:38:47.333861+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:38:47.339545+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:41:06.698445+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:41:06.702100+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:41:31.984808+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:41:31.991373+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:41:41.538569+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:42:30.299440+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-13 17:42:30.302444+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-224]
2024-05-20 17:22:56 database: wrong user or password; user: "AKOROVAI"
org.h2.message.DbException: Nieprawidłowy użytkownik/hasło
Wrong user name or password [28000-220]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.Engine.openSession(Engine.java:154)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at com.intellij.database.remote.jdbc.helpers.JdbcHelperImpl.connect(JdbcHelperImpl.java:768)
	at com.intellij.database.remote.jdbc.impl.RemoteDriverImpl.connect(RemoteDriverImpl.java:153)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nieprawidłowy użytkownik/hasło
Wrong user name or password [28000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:522)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 28 more
2024-05-20 17:23:10 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:23:10 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:26:51 jdbc[25]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:27:13 jdbc[26]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:43:12 jdbc[117]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:43:19 jdbc[128]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:45:55 jdbc[4]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:45:55 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:45:55 jdbc[5]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:48:21 jdbc[26]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:48:24 jdbc[27]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:49:30 jdbc[38]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
2024-05-20 17:49:45 jdbc[49]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:712)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:587)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:828)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:705)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:704)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
